systemd/ipa-healthcheck.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+LOGDIR=/var/log/ipa/healthcheck/
+DATE=$(date +%Y%m%d)
+
+/usr/bin/ipa-healthcheck --output-file $LOGDIR/healthcheck.log-$DATE
```

Comment:
nitpick: LOGDIR should be /var/log/ipa/healthcheck (no trailing /)
and line 5 should end with ${LOGDIR}/healthcheck.log-$DATE

Otherwise the command is constructed as "usr/bin/ipa-healthcheck --output-file /var/log/ipa/healthcheck//healthcheck.log-20190425" (double /)